ebos/ecloutputblackoilmodule.hh

```diff
@@ -830,6 +844,14 @@ public:
         if (bubblePointPressure_.size() > 0)
             sol.insert ("PBUB", Opm::UnitSystem::measure::pressure, std::move(bubblePointPressure_), Opm::data::TargetType::RESTART_AUXILIARY);
 
+
+        if (swMax_.size() > 0)
+            sol.insert ("SWMAX", Opm::UnitSystem::measure::identity, std::move(swMax_), Opm::data::TargetType::RESTART_AUXILIARY);
+
+        if (porvMultiplier_.size() > 0)
+            sol.insert ("PORV_MOD", Opm::UnitSystem::measure::identity, std::move(porvMultiplier_), Opm::data::TargetType::RESTART_AUXILIARY);
+
```

Comment:
@joakim-hove I expected this to be sufficient to add PORV_MOD to the restart file, but it isn't. Do I need to modify something in opm-common or libecl?

Reply:
I'll look into it - what you have done *should* be enough, but ....

Reply:
Changing it to RESTART_SOLUTION seems to work.

Reply:
> Changing [the `target`] to `RESTART_SOLUTION` seems to work.

Are you emitting strictly ECLIPSE-compatible restart files (`--enable-opm-rst-file=false`)?  If so, the vectors marked as `RESTART_AUXILIARY` get [dropped on the floor](https://github.com/OPM/opm-common/blob/7a2c1157a02259cad5deed554db08877f5af5b89/src/opm/output/eclipse/RestartIO.cpp#L499-L501)

Reply:
@bska Thanks for the explaination.

Reply:
I guess it would be good if the output code would at least produce a warning in that case?